### PR TITLE
Delete-sku-pack minor enhancement

### PR DESCRIPTION
**Pull request by uppalk1**
Check if sku pack info is deleted from Skus (get-sku) after a delete sku pack

### CODE REVIEW

**Comment by anhou** [comment]
:+1:

File: test/tests/api/v2_0/skupack_tests.py [hunk 1]
```diff
@@ -244,13 +242,29 @@ def post_skupack(self):
...
                     assert_equal(200, result.status, message=result.reason)
-
+                    data = loads(result.data)
+                    LOG.info("ID of the deleted sku pack is:   "+ data.get('id'))
+                    Api().skus_get()
+                    skus = loads(self.__client.last_response.data)
+                    LOG.info("List of Skus after delete sku pack")
+                    LOG.info(skus)
+                    assert_equal(0, len(skus))
+                    for n in skus:
```
Comment (yyscamper) [inline test/tests/api/v2_0/skupack_tests.py]: Seems this for loop is useless due to the above assertion, as the skus length will always be zero here.

File: test/tests/api/v2_0/skupack_tests.py [hunk 2]
```diff
@@ -244,13 +242,25 @@ def post_skupack(self):
...
                     assert_equal(1, len(loads(self.__client.last_response.data)))
-
+                    
                     """Test DELETE:api/2.0/skus/:identifier/pack"""
+                    Api().skus_get()
+                    skus = loads(self.__client.last_response.data)
+                    LOG.info("Printing Sku from get skus before a delete sku pack")
+                    LOG.info(skus)
                     Api().skus_id_delete_pack(self.__packFolderId)
                     Api().skus_id_delete(self.__packFolderId)
```
Comment (zyoung51) [inline test/tests/api/v2_0/skupack_tests.py]: Need an additional GET after the pack delete to validate the sku pack related keys are correctly deleted from the sku object.

Comment (uppalk1) [inline test/tests/api/v2_0/skupack_tests.py]: @zyoung51  Line 257 has it, is that what you mean ?

**Comment by yyscamper** [comment]
:+1:

File: test/tests/api/v2_0/skupack_tests.py (hunk 2 above)
Comment (zyoung51) [inline test/tests/api/v2_0/skupack_tests.py]: @uppalk1 Part of the request was adding a check between lines 251 and 252 to validate that the delete pack did what it is supposed to do.  The check at 257 validates that 252 (delete) did something, but does not validate that 251 (delete pack) properly removed the sku pack related keys.

Comment (uppalk1) [inline test/tests/api/v2_0/skupack_tests.py]: @zyoung51 Done, Thank you.

**Comment by uppalk1** [comment]
test this please !

**Comment by zyoung51** [comment]
+1

**Pull request merged**